docs/Reference/Request.md

```diff
@@ -90,6 +92,7 @@ fastify.post('/:params', options, function (request, reply) {
   console.log(request.protocol)
   console.log(request.url)
   console.log(request.routerMethod)
+  console.log(request.routeBodyLimit)
```

Comment:
Where does this come from? Wouldn't it actually be:

```suggestion
  console.log(request.routeOptions.bodyLimit)
```

Reply:
done 👍